egs/jsut/tts1/local/data_prep.sh

```diff
@@ -15,19 +16,11 @@ utt2spk=${data_dir}/utt2spk
 spk2utt=${data_dir}/spk2utt
 rawtext=${data_dir}/rawtext
 text=${data_dir}/text
-usr_dict_dir=/usr/local/lib/mecab/dic/mecab-ipadic-neologd
-esp_dict_dir=../../../tools/mecab/mecab-ipadic-neologd
 
 # check file existence
 [ -e ${scp} ] && rm ${scp}
 [ -e ${utt2spk} ] && rm ${utt2spk}
 [ -e ${rawtext} ] && rm ${rawtext}
-if [ ! -e ${usr_dict_dir} -a ! -e ${esp_dict_dir} ]; then
-    echo "$0: Error: The mecab-ipadic-NEologd can not be found" >&2
-    echo "$0: Error: Please modify the dictionary path in this script or" >&2
-    echo "$0: Error: install it by running 'make mecab.done' at espnet/tools" >&2
-    exit 1
-fi
 
 # make scp, utt2spk, and spk2utt
 find ${db} -name "*.wav" | sort | while read -r filename; do
```

Comment:
To find in symlink, better to use `-follow` option.
```suggestion
find ${db} -follow -name "*.wav" | sort | while read -r filename; do
```